index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="author" content="iamsrk">
```

Comment:
Please  remove author meta tag

Reply:
Why remove ? What's harm in it ?

Reply:
This massage was only for  understanding  purpose how we work on pr request @faran4engg  Sir